tests/integration/test_storage_filelog/test.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+import time
+import shutil
+import os
+from helpers.client import QueryRuntimeException
+from helpers.cluster import ClickHouseCluster
+
+cluster = ClickHouseCluster(__file__)
+
+node1 = cluster.add_instance('node1',
+                             main_configs=['configs/user_files_path.xml'])
+
+
+@pytest.fixture(scope="module")
+def started_cluster():
+    try:
+        cluster.start()
+        yield cluster
+
+    finally:
+        cluster.shutdown()
+
+
+def test_select(started_cluster):
+    try:
+        name = "filelog_table"
+        node1.query("""
+            CREATE TABLE {name} (id UInt64, value UInt64) Engine=FileLog('logs', 'CSV')
+        """.format(name=name))
+
+        assert node1.query("SELECT sum(id) FROM {name}".format(name=name)) == "55\n"
+
+        shutil.copy("user_files/logs/data.csv", "user_files/logs/data_copy.csv")
+
+        assert node1.query("SELECT sum(id) FROM {name}".format(name=name)) == "55\n"
+
+    finally:
+        node1.query("DROP TABLE IF EXISTS {name}".format(name=name))
+        os.remove("user_files/logs/data_copy.csv")
+
+
+def test_stream_to_view(started_cluster):
```

Comment:
Did you try to run these tests locally? With runner or pytest.
btw, I think if you want you can make functional bash test instead. Whatever is easier to you.

Reply:
If write functional test, which places can be used to put log files?

Reply:
See for example how it is done in `01889_sqlite_read_write.sh`.